### DISCUSSION TITLE
v41.0

### DISCUSSION THREAD

**Discussion by nebutech-admin** [NebuTech/NBMiner#808] [General]
`feature`: `ethash` 100% LHR unlocker added, for both Windows & Linux.

- Run nbminer with admin priviledge to get 100% LHR unlock
- Tested and verified on drivers: `512.15` for Windows, `510.60` for Linux.
- Other driver versions may have some compatibility issue.
- Previous LHR mode are removed.

<hr /><em>This discussion was created from the release <a href='https://github.com/NebuTech/NBMiner/releases/tag/v41.0'>v41.0</a>.</em>

**Comment by VIONICK08** [comment]
3080 12gb? 3050?

**Reply by l-Kage-l** [reply]
HiveOS version when?

**Reply by BIGguiX** [reply]
3080 12gb and 3050 have a diffrent lhr version.  doubt they'll be unlocked

**Reply by suzwon** [reply]
cd /tmp && wget https://github.com/NebuTech/NBMiner/releases/download/v41.0/NBMiner_41.0_Linux.tgz && tar -xvf NBMiner_41.0_Linux.tgz && cd NBMiner_Linux && miner stop && cp nbminer /hive/miners/nbminer/40.1 && miner start

**Reply by naman09i** [reply]
> cd /tmp && wget https://github.com/NebuTech/NBMiner/releases/download/v41.0/NBMiner_41.0_Linux.tgz && tar -xvf NBMiner_41.0_Linux.tgz && cd NBMiner_Linux && miner stop && cp nbminer /hive/miners/nbminer/40.1 && miner start

not able to do so ..after putting the command it agains start 40.1 ? can you explain step by step

**Reply by nieXas** [reply]
> > cd /tmp && wget https://github.com/NebuTech/NBMiner/releases/download/v41.0/NBMiner_41.0_Linux.tgz && tar -xvf NBMiner_41.0_Linux.tgz && cd NBMiner_Linux && miner stop && cp nbminer /hive/miners/nbminer/40.1 && miner start
> 
> not able to do so ..after putting the command it agains start 40.1 ? can you explain step by step

set the miner to 40.1 instead of the lastest, then retry command

**Reply by 4DRigMiner** [reply]
Estou tendo problemas, fans vai a zero e mineração cai pela metade, alguém sabe resolver? Obrigado
![quedanbminer](https://user-images.githubusercontent.com/105185477/167333416-30aaeb61-9ad2-4469-b617-9b1937ff6811.jpg)

**Comment by ansonk4** [comment]
Any command for update on hive?

**Reply by MarianoAlv** [reply]
Just tried manually but it failed, anyone have succesfully install on hiveos?

**Reply by SOTLx** [reply]
miner stop && cd /tmp && wget https://github.com/NebuTech/NBMiner/releases/download/v41.0/NBMiner_41.0_Linux.tgz && tar -xvf NBMiner_41.0_Linux.tgz && cp /tmp/nbminer /hive/miners/nbminer/40.1/ && miner start

**Reply by magrhino** [reply]
Also make sure you run nbminer version 40.1 before you run that command^ to ensure it's downloaded on your rig

**Reply by MarianoAlv** [reply]
you are awesome, testing now.

**Reply by MarianoAlv** [reply]
Just tried but gettin: cannot stat ´/tmp/Nbminer´: no such file directory

**Reply by SOTLx** [reply]
cd /tmp && wget https://github.com/NebuTech/NBMiner/releases/download/v41.0/NBMiner_41.0_Linux.tgz && tar -xvf NBMiner_41.0_Linux.tgz and copy file nbminer to folder /hive/miners/nbminer/40.1 with replace. use mc command in hive shell

**Reply by coodeLuck** [reply]
Thanks it's working! My RTX 3080 LHR gives 98 MH, was 79 MH

**Reply by suzwon** [reply]
cd /tmp && wget https://github.com/NebuTech/NBMiner/releases/download/v41.0/NBMiner_41.0_Linux.tgz && tar -xvf NBMiner_41.0_Linux.tgz && cd NBMiner_Linux && miner stop && cp nbminer /hive/miners/nbminer/40.1 && miner start

**Reply by MarianoAlv** [reply]
![image](https://user-images.githubusercontent.com/60038865/167301412-f3d7f913-1d6d-4881-a3dd-11e514c8cc23.png)

Thank people this is awesome!!

**Reply by Siehen** [reply]
How you get the problem fixed wioth tmp nbminer. Also Server on Hive down, cant Update Driver

**Reply by d2Dreamer** [reply]
> How you get the problem fixed wioth tmp nbminer. Also Server on Hive down, cant Update Driver

nvidia-driver-update https://us.download.nvidia.com/XFree86/Linux-x86_64/510.60.02/NVIDIA-Linux-x86_64-510.60.02.run

**Reply by AdiHash82** [reply]
Hey Guys, Im unable to update on HiveOS, when I run the command its downloading the miner but unable to instal. Please help.


2022-05-08 20:39:24 (8.87 MB/s) - ‘NBMiner_41.0_Linux.tgz’ saved [10315775/10315775]

NBMiner_Linux/nbminer
NBMiner_Linux/nbminer.sha256
NBMiner_Linux/readme.md
NBMiner_Linux/start_ae.sh
NBMiner_Linux/start_beam.sh
NBMiner_Linux/start_config.sh
NBMiner_Linux/start_conflux.sh
NBMiner_Linux/start_ergo.sh
NBMiner_Linux/start_etc.sh
NBMiner_Linux/start_eth.sh
NBMiner_Linux/start_rvn.sh
No miner screens found

**Reply by AdiHash82** [reply]
Ok, sorted now. Up and runing 41.0 Ver. RTX 3080 LHR running 100.9 MH\s straight away. Niiiiice, thanks guys.

**Reply by amtmathauda** [reply]
> > How you get the problem fixed wioth tmp nbminer. Also Server on Hive down, cant Update Driver
> 
> nvidia-driver-update https://us.download.nvidia.com/XFree86/Linux-x86_64/510.60.02/NVIDIA-Linux-x86_64-510.60.02.run

It's not working for my rig. Pls help.

**Reply by amtmathauda** [reply]
Ok, sorted now. Up and running 41.3 Ver. RTX 3060 Ti LHR running 58 MH\s on Hiveos.

**Comment by sunyao920807** [comment]
Looking forward to a low power model！ ts

**Reply by xPatrikTwitch** [reply]
I dont think there will be a low power mode. This is no lhr and same as FHR gpus. Just drop ur clocks and that way you can get lower power

**Comment by BreakingGalaxy** [comment]
Hi cool stuff, and thanks for the update but some lhr gpu are not detected as such is there a way to force it? cus i cant find it in the documentation, it doesnt even show up on the console that the lhr unlock was aplied
![image](https://user-images.githubusercontent.com/105158722/167298011-cb5b4d5a-0cdf-4029-8677-4fec0cf15c87.png)
![image](https://user-images.githubusercontent.com/105158722/167298044-2943353d-a199-4dfd-b614-34930ca1a238.png)

**Reply by BreakingGalaxy** [reply]
i reinstalled the drivers reseted the gpus and is working great now

**Comment by YAIBDK** [comment]
I open on Windows 11 but v41.0 show "access denied" in command line
when im back to v40.1 it's work normally.

Please help. Thanks.

**Reply by suzwon** [reply]
Run as Administrator

**Reply by YAIBDK** [reply]
it still the same, thanks.

**Reply by Tom5051** [reply]
screenshot?

**Reply by YAIBDK** [reply]
problem solved !

becauase my windows 11 did not show "windows defender"
i had to fix it by [https://answers.microsoft.com/en-us/windows/forum/all/windows-defender-broken-on-windows-11-and-wont/c0217d1b-51e7-4323-a003-76220e089eb4](url)

and i add "nbminer.exe" exclusion in windows defender

after that i ran v41.0 as admin

thanks

**Reply by Tom5051** [reply]
Yep, don't forget about that pesky Windows Defender exclusion :)

**Comment by Spicymufinhd** [comment]
using 510.60.02 driver in a mixed AMD/Nvidia rig, can't go past DAG verification on the new 41.0 in HiveOs

**Reply by Jimweb3** [reply]
same

**Reply by orcunver** [reply]
same error code 11 restart and restart

**Reply by Tom5051** [reply]
Saw another post that claimed to fix this by running only Nvidia cards on nbminer and running either a second instance for AMD or a different miner.

**Reply by RuaGulu** [reply]
Same here with error code 11.

**Reply by nebutech-admin** [reply]
issue confirmed, will be fixed in the next release.
You can run seperate miner for AMD (use old version) and Nvidia for now.

**Reply by magrhino** [reply]
I have it working on my mixed rig, try running a different miner for your AMD/FHR cards, and put the LHR on nbminer
![image](https://user-images.githubusercontent.com/12738722/167497632-ed829af5-acfe-4348-a9df-bdf94967a5db.jpeg)

**Comment by Suren2022** [comment]
как запустить от имени админа? щелкаю правой кнопкой  мышки на бат файл и нажимаю запустить от имени админа потом появляется командная строка а в ней надпись нажмите любую клавишу

**Reply by ghost** [reply]
Измените команду bat, чтобы указать путь к папке. Или добавьте путь к системной переменной path. (Гугл переводчик, может быть не точен)

**Comment by Tom5051** [comment]
Testing now. Looks very good so far. Great job with the LHR unlock, you are legends!

**Comment by Tom5051** [comment]
![image](https://user-images.githubusercontent.com/58039085/167299301-a404db51-d219-4a38-9368-0a94ba082f2a.png)
![image](https://user-images.githubusercontent.com/58039085/167299120-8575b856-4bb1-4a37-9ca2-f7a37f5bd3c2.png)

was getting:
3060ti - 48mh/s @ 141W
3070ti - 60mh/s @ 195W
3080ti - 92mh/s @ 286W
THANK YOU!!!

**Reply by USCMig** [reply]
would you mind sharing what your .bat file looks like? I don't typically use nbminer but I'm not setting it up correctly apparently?  Also how do you setup the web api stuff?

**Reply by Tom5051** [reply]
web api is setup by default. Just browse to http://127.0.0.1:22333 on your mining rig.

**Comment by b0she** [comment]
![image](https://user-images.githubusercontent.com/103332525/167300362-d47db891-9fc0-4f09-ab64-f5b87931a182.png)
It works on my 3070, thank you!

**Comment by phantasina** [comment]
Can the 3060 12g unlockable?

**Reply by kelib99** [reply]
`RTX 3060 LHRv1, I installed 512.59 win drivers and hashrate dropped from 40Mh/s to 22 Mh/s`

Edit: Even **with new drivers working well** (I didnt run miner as an administrator (facepalm)

**Reply by b0she** [reply]
512.15 Install and try

**Reply by Tom5051** [reply]
Isn't the 3060 v1 already unlocked with driver 470.05

**Reply by kelib99** [reply]
@Tom5051 Yes, it is. But it must be on Windows and on full PCIe slot.

**Reply by Tom5051** [reply]
> @Tom5051 Yes, it is. But it must be on Windows and on full PCIe slot.
Thanks, I didn't know about the full slot requirement.

**Reply by samhaswon** [reply]
On Linux, my MSI Ventus 3x RTX 3060 LHR V2 works well (Zotac GTX 1660 also, but no improvement). lolMiner's raw hashrate for it was ~49MH/s and I get about 47-48 with this unlock. This is up from 37.1MH/s on lolMiner and similar on T-Rex miner, so I call this an absolute win! I lost the silicon lottery and can only offset the memory by 2200/1100 (Linux/Windows), but I'll take what I can get.

**Comment by NVRFLZ** [comment]
![Снимок экрана 2022-05-08 164707](https://user-images.githubusercontent.com/43913012/167300828-41f86054-cac7-46c9-ab6a-d0408362d925.png)

3080 12gb, not bad thanks!

**Comment by Apollograph** [comment]
![image](https://user-images.githubusercontent.com/105162081/167301306-d54bbe5c-36ce-462d-a269-676d6becedbd.png)
dont work(

**Reply by diguoemo** [reply]
you must use 512.15

**Reply by Apollograph** [reply]
> you must use 512.15

no 
510.60.02
2 cards in rig 6800 + 3060Ti

**Reply by kaitranntt** [reply]
Can't mix LHR with AMD card.
I use TeamRedMiner for the AMD one, and NBMiner for the rest LHR

**Reply by naman09i** [reply]
> Can't mix LHR with AMD card.
> I use TeamRedMiner for the AMD one, and NBMiner for the rest LHR

are you running on hive or windows ?

**Reply by Apollograph** [reply]
> > Can't mix LHR with AMD card.
> > I use TeamRedMiner for the AMD one, and NBMiner for the rest LHR
> 
> are you running on hive or windows ?

hive os bro

**Comment by sancak677** [comment]
![Screenshot_2022-05-08-17-18-57-116_hiveos mobile](https://user-images.githubusercontent.com/45976643/167301514-ce83966c-0cac-46f9-83cc-253d81fd9be3.jpg)

64mhs aldım

**Reply by Tom5051** [reply]
update to the latest version of HiveOS

**Comment by amituofo99** [comment]
Having 2 3060 LHR V1 mixed with 1 6700 XT and 1 6900 XT
Running on latest 510.60.02
Code 11 error from NBminer
miner reboot loop
![image](https://user-images.githubusercontent.com/105163188/167302384-cf225f67-36b1-4b2d-9010-42b67998a10c.png)

**Reply by Siehen** [reply]
Use different Miner for AMD Cards. Its now not compatible with AMD

**Reply by amituofo99** [reply]
> Use different Miner for AMD Cards. Its now not compatible with AMD

THANK  YOU! IT WORKS NOW! 
Used LOLminer for AMD cards
Nbminer for the 3060 LHR cards :) Thank you!

**Comment by Gloom536** [comment]
Have you been able to use this new miner update with RaveOS? Latest Nvidia Driver on RaveOS is 510.54 (so 510.60 is not available, or I don't know how to install it!). It seems to just not be compatible at all. I have 2 rigs. On one rig the miner work, but hashrate are really low. On the other rig, the miner crash after few second. Saying unexpected exit of the miner (I tried to remove mem overclock to see if it was my overclock, but no).

So, do you know if we can use this miner update as of right now with RaveOS? Have you tried on an older Nvidia update which would work? Or were you able to update Nvidia Driver to 510.60 on RaveOS?

**Reply by amituofo99** [reply]
Sorry I am only using HiveOS for now! hahaha

**Reply by sunnyphp** [reply]
Go to rig console and update to latest OS version:
```shell
upgrade -b 8820-2060
```
My rig hashrate increased from ~500 Mh/s to ~650 Mh/s. Works fine.

**Reply by Gloom536** [reply]
> Go to rig console and update to latest OS version:
> 
> ```shell
> upgrade -b 8820-2060
> ```
> 
> My rig hashrate increased from ~500 Mh/s to ~650 Mh/s. Works fine.

How do you know that you have to write those number? Is there a website somewhere that tell which "numbers" = which nvidia version?

**Reply by Gloom536** [reply]
I have two rigs, one worked with the version you told me to install. But the second one always does this after 30 sec maximum. Even if all my mem clock are at +0mhz. 
![image](https://user-images.githubusercontent.com/105163385/167320557-8657a61f-b0ad-40b5-abf4-2c3d652bb0f5.png)

**Comment by Mustangboy94** [comment]
I have no issues on Trex with 4 gpu running kicked them over to NBminer and get error below. when i have 3 gpu its fine add the 4 back in i get this. I am using Hiveos with the right driver.
![image](https://user-images.githubusercontent.com/105165989/167305042-e91df900-b323-4546-8dcc-394bd1ef72c5.png)

**Reply by naman09i** [reply]
what driver are you using ?

**Reply by Mustangboy94** [reply]
I am using 510.60 for Linux

**Reply by naman09i** [reply]
i am using 510.68 its working fine, you can try too

**Reply by kaitranntt** [reply]
If your rig is a mixture of LHR with AMD gpus, use other miner for the AMD ones

**Reply by Mustangboy94** [reply]
Nope all Nvidia 70x2 80x2

**Reply by Mustangboy94** [reply]
I tried that driver also (510.68)

**Comment by tomcruise946** [comment]
Cards are crashing any bugs r there

**Reply by cybeattack** [reply]
Maybe they fix soon

**Comment by tomcruise946** [comment]
Bugs

**Comment by tomcruise946** [comment]
Gpu driver error hiveos

**Reply by Tom5051** [reply]
what driver version, what cards are you running?

**Comment by lucasfigueiredoweb** [comment]
After 15-20min reboot a random card gets fan 0% and hashrate drops.

Using latest drivers.

Sometimes 3070ti, sometimes 3060, sometimes 3060ti..

![image](https://user-images.githubusercontent.com/12434944/167307980-d8bcbd30-9734-4379-8704-d73ea6f7342a.jpeg)

**Reply by ansonk4** [reply]
I have the same issue

**Reply by Tom5051** [reply]
does it do this with stock OC settings?

**Reply by chechle39** [reply]
I have the same issue

**Reply by naman09i** [reply]
same issues ia m trying to change the drivers to 510.60.02 but there is some connectivity issues coming on hiveos probably because everyone would be trying to update the setting on hive os and there might be server is fully loaded.

**Reply by senadbrkic** [reply]
510.60.02 has same behavior

![image](https://user-images.githubusercontent.com/2125057/167327261-dd41ec7c-b9d3-4b56-8417-4935dedcaed9.jpeg)

**Reply by RuaGulu** [reply]
Got the same issue with 3070ti core 1100 and memory +2200 here.

**Reply by 4DRigMiner** [reply]
I have same problem
![quedanbminer](https://user-images.githubusercontent.com/105185477/167333715-fdd80903-c046-48ad-b87b-bdc8944e27da.jpg)

**Reply by senadbrkic** [reply]
now affecting 3070ti and 3080ti on other rigs

**Comment by leoli-dev** [comment]
<img width="722" alt="Screen Shot 2022-05-08 at 12 58 59 PM" src="https://user-images.githubusercontent.com/1598600/167308678-f8e0a3d3-0f23-4956-adfe-1fae03211632.png">

This version doesn't work correctly with my 3070ti (mining on HiveOS), the hashrate has been reduced from 60M/s to 43M/s

**Reply by Tom5051** [reply]
your power level is too low. 196watts works best for the 3070ti but you can bring the value down until you have a satisfactory hash rate vs power usage

**Reply by leoli-dev** [reply]
> 

It seems not a power level problem, I tried increase it, but it doesn't help

<img width="634" alt="image" src="https://user-images.githubusercontent.com/1598600/167315892-6562466c-b34c-4b37-a9cb-77002ca8e67d.png">

**Reply by Tom5051** [reply]
Memory usage should be 100% shouldn't it? Weird you're getting 66%
Did you update your nvidia drivers to 510.60?
Intensity settings?

**Reply by leoli-dev** [reply]
> Memory usage should be 100% shouldn't it? Weird you're getting 66% Did you update your nvidia drivers to 510.60? Intensity settings?

Yes, you're right! I haven't update the nvidia driver since a long time, now I got the latest one, and LHR is no more LHR, my 3070ti is doing very great, thank you!

<img width="752" alt="Screen Shot 2022-05-08 at 11 21 00 PM" src="https://user-images.githubusercontent.com/1598600/167335567-80cda637-b1d9-4c61-b918-941ebe468e48.png">

**Comment by SantaAngel** [comment]
Good afternoon.
When I try to start the miner, I get the following error. I tried everything. Driver 512.15, run as administrator, stock video card settings. I tried to run the previous version of the miner and other algorithms - the same error.
Windows 10. 2x3080TI & 750TI (for monitor).
![image](https://user-images.githubusercontent.com/105172296/167312253-7c67036b-cc69-4bea-9392-4d63f1325c6f.png)

**Reply by Tom5051** [reply]
Stick to 40.1, you won't gain any benefits froom 41.0 mining raven. It will only increase ETH mining on LHR cards

**Reply by SantaAngel** [reply]
> 
It was the text and error code that occurs on all algorithms. Here is an ETH example

![image](https://user-images.githubusercontent.com/105172296/167367114-a29156d3-bf42-4b44-b973-a6cd887186be.png)

**Comment by kirkhilles** [comment]
![NBminer](https://user-images.githubusercontent.com/1503360/167312480-be97cb2a-7557-4e23-93f0-6a5bd0cd7c81.png)

I was like "Why isn't this updating! It's still 40.1" then I later saw that they hadn't updated it yet. Makes sense as I'm sure it was an all-nighter to get it done.

**Comment by Manuelgarcia19891109** [comment]
No puedo hacer que funcione con las GPU AMD 😭

**Reply by JuanCruzLopezz** [reply]
No funciona para las AMD, solo para las Nvidia

**Comment by MarianoAlv** [comment]
imposible to installa drivers, server is freeze

**Comment by Aram-Badalyan** [comment]
> `feature`: `ethash` 100% LHR unlocker added, for both Windows & Linux.
> 
> * Run nbminer with admin priviledge to get 100% LHR unlock
> * Tested and verified on drivers: `512.15` for Windows, `510.60` for Linux.
> * Other driver versions may have some compatibility issue.
> * Previous LHR mode are removed.
> 
> _This discussion was created from the release [v41.0](https://github.com/NebuTech/NBMiner/releases/tag/v41.0)._

The miner works for 30 minutes and turns off some cards on Ubuntu 18.04. Driver version: 510.60. There are no such problems with other miners. What could be the cause of the problem? Tried to reduce overclock, doesn't help.

**Reply by chechle39** [reply]
I have the same issue

**Reply by Tom5051** [reply]
If you're running a mix of AMD and nvidia, try running the AMD cards on a different miner program

**Reply by Aram-Badalyan** [reply]
> If you're running a mix of AMD and nvidia, try running the AMD cards on a different miner program

Only Nvidia

**Comment by Manuelgarcia19891109** [comment]
Si muchas gracias

**Comment by Manuelgarcia19891109** [comment]
Si 510.60.02 y nbminer, teamredminer
Overlock un poco más bajo memorias
Anteriormente lolminer 1.49

**Comment by mohsenk94** [comment]
Guys with stability issues (Mostly fan hitting 0 on a GPU - hashrate locking at 1/3 capacity, GPUs falling out of order etc..)
Please participate in this thread to see how we can fix it and contribute to future improvements
https://github.com/NebuTech/NBMiner/issues/825

**Reply by 4DRigMiner** [reply]
its same
![quedanbminer](https://user-images.githubusercontent.com/105185477/167333546-4c04fce0-ee07-457c-b421-2173497c01e5.jpg)

**Reply by caramelzane** [reply]
Try rollback drivers to 510.60 and update hive to latest version and change flight sheet to NBMiner Latest instead of overwriting v40.1 with v41.

CC: 1500
MC: 2200
PL 120 (For safety)

Spent the whole of last night tuning and changing drivers.

I can confirm Nvidia Drivers earlier than 510.60 does not work with the NBMiner LHR 100% Unlock.

**Reply by xMAXH0x** [reply]
Наблюдаются такие же проблемы с произвольным отвалом карт
драйвер корректный, версия последня вроде. пробовал поднимать ПЛ, не помогает. 
![image](https://user-images.githubusercontent.com/105199081/167365009-a6c3fc0b-80be-4ae3-b670-9d61476e696d.png)

**Comment by basking-in-the-sun2000** [comment]
Doesn't seem to report the hashrate to the pool. Crazypool shows current speed and shares, but not a self-reported value

**Comment by Jimweb3** [comment]
`[03:21:05] INFO - Device 3: DAG - Built, 6.63 s.
[03:21:06] INFO - Device 2: DAG - Built, 11.06 s.
[03:21:06] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
[03:21:06] ERROR - Mining program unexpected exit.
[03:21:06] ERROR - Code: 11, Reason: Process crashed
[03:21:06] ERROR - Restart miner after 10 secs ...
[03:21:06] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!`


Amd580 * 3 and 3070Ti *1 3080 * 1

**Reply by mycevher** [reply]
3060ti afterburner windows ayarı verebilirmisiniz

**Comment by mycevher** [comment]
Arkadaşlar windows 3060ti ve 3070ti afterburner ayarlarını verebilirmisiniz sorun yaşıyorum bilgisayar çöküyor mem1300 core 400de bütün kartlarımda aynı olay oluyor

**Comment by mintbird** [comment]
👍 My 3060ti (PNY Uprising) got 60.58 Mh/s

![image](https://user-images.githubusercontent.com/695425/167354526-adb152ff-6067-48d4-ad99-874e82c64e37.png)

**Reply by mycevher** [reply]
Mem 8000mi olacak yazarsanız sevinirim ayarlari inanın birșey anlamadım. Afterburner ayarlarını atarsanız resimli çok sevinirim. Gruptaki herkes sayenizde faydalanır.

**Reply by mintbird** [reply]
Locked core clock at 1380 MHz, memory clock +1200 (windows), Power limit does not set.

NBminer cmd parameters is here
`-cclock @1380 --mclock 1200`

**Comment by rztprog** [comment]
![FSTa4Q9X0AAL-58](https://user-images.githubusercontent.com/45171753/167376070-e791dedb-dec0-4077-a923-418ba1cd5c60.png)

Thanks ! :)

**Comment by mycevher** [comment]
3060ti yada 3070ti afterburner ayarları yokmu sizde windows icin

**Comment by M0TFF** [comment]
Can someone please tell me what clock settings I should use for a RTX 3080 Ti.

**Reply by amusleh-spotware-com** [reply]
![Screenshot_3](https://user-images.githubusercontent.com/8859902/167397408-e2b42849-2ed0-456c-83bf-7c56e4df33a5.png)

**Reply by j1mb0b** [reply]
![Screenshot 2022-05-09 135255](https://user-images.githubusercontent.com/15685798/167404534-d2f2d8db-92e1-4403-b3cf-59aeaa3202f6.png)

**Reply by M0TFF** [reply]
Are you using MSI afterburner?

**Reply by rztprog** [reply]
![image](https://user-images.githubusercontent.com/45171753/167428621-0d4e6e12-cdce-49b7-9f2a-b30db759392d.png)

**Comment by rckn333** [comment]
dual 3060s, I was testing start_eth to see what I'd get upgrading from an earlier version.  I accidently clicked to Start_eth_overclock with the default settings in it, settings kicked in and fans all went to 100%.  I closed out the batch but the OC settings remaining, went into afterburner and could not adjust anything.  Restarted and it went right back into the same OC settings that I can not adjusted?

What can I do to get back control?

**Comment by Stanley20222** [comment]
Hello. I have the following problem. Stably once every 30 minutes, mining is reset for 1 second, which I learn from a wifi outlet that notifies of a power drop. The crash happens on all cards at once (nVIDIA) and not just on the LHR version. At the same time, the miner does not show that there was a drop in power or hashrate. Such falls have a very negative effect on the operation of video cards. What can you suggest? Thank you.
![image](https://user-images.githubusercontent.com/105218319/167416031-e8155966-205e-44df-8966-ff99f3785abe.png)
![image](https://user-images.githubusercontent.com/105218319/167416064-4171e38e-a74c-4588-bbd5-7ed6a27cae33.png)

**Reply by s0laria** [reply]
try --no-interrupt ?

**Reply by Stanley20222** [reply]
Thank you. I'll try

**Comment by ilker6** [comment]
good day, my 3 pcs msi suprim x rtx 3080ti cards are 200 cores, 2000 memory and at max 92/94 degrees temperatures to keep the temperature constant, pl 300 - 220 -230w values, although I do not experience resets or similar situations in other software, the fan value is 0 and the hash value at random times " It drops to /3".
  It does not return to regular operation without turning the system off and on. Still, the LHR breaking process was great, I wish you success and I'm sure you can solve these small problems.

**Reply by amusleh-spotware-com** [reply]
I have the same issue

**Reply by anarki1302988** [reply]
Sama Here on 3060ti and 3080s, while 3080ti is getting driver error, even tho using 510.60 and 510.68.
It's not stable at all, can't run a day without issues.

**Comment by caramelzane** [comment]
I hope I am not jumping the gun. However, I kinda fixed the unstable issue with 3060 V2.
![Greenshot 2022-05-09 22 01 34](https://user-images.githubusercontent.com/105192388/167428483-2020d28d-0498-487b-8624-a0e05bd7c9ec.png)
Try Negative Core.

My Settings are:

CC: -500
MM: 2200
PL: 102 - 110 (increment of 1w to get desired hash rate.

**Comment by s0laria** [comment]
I have Nvidias 3060Ti only, 510.60 driver, Ubuntu 18.04 system.
After about 10 minutes mining on stock memory clocks I'm getting:

ERROR - CUDA Error: unspecified launch failure (err_no=4)
ERROR - Device X exception, exit ...

After the next miner start that GPU remains on 1/3 hashpower, until next reboot.

**Reply by s0laria** [reply]
OK, further lowering mem clocks helps. After lowering mem -100MHz it's stable for 30 minutes now.

**Reply by Aram-Badalyan** [reply]
The same problem, it works for 30 minutes and the hashrate drops from 360 to 120. Restarting the miner does not help, only rebooting.

**Reply by s0laria** [reply]
With lower mem clocks stable around 1 hour, but this error always pops-up on some card sooner or later:

ERROR - CUDA Error: unspecified launch failure (err_no=4)

The miner autorestarts, but that GPU remains on 1/3 hashpower until reboot. I guess the only stable operation for me is to force autoreboot every 30 minutes or so.

**Comment by tovimanjer** [comment]


**Reply by mycevher** [reply]
Aynı sıkıntı bende var. Çözüm bulamadım

**Comment by Tonyx233** [comment]
Can someone share me what clock settings I should use for a RTX 3070 Ti. Thanks!!

**Reply by ghost** [reply]
I have a couple on a HiveOS rig using these settings:
Core 1275
Mem 1650
PL 220
I'm getting 76-78Mh/s on each, pretty stable I must say

**Reply by Tom5051** [reply]
core - 0 (not sure why people overclock the GPU core, does nothing on my rigs)
mem - 1550
PL - 196 or 67%
Fan - 100%
82MH/s all day long ETH

**Reply by Tom5051** [reply]
![image](https://user-images.githubusercontent.com/58039085/167474320-0a326200-83e6-4146-90a1-33549c1c35de.png)

**Comment by MarianoAlv** [comment]
i just can not update drivers, any other link from where i can download?

**Reply by MarianoAlv** [reply]
This one worked:
>nvidia-driver-update https://us.download.nvidia.com/XFree86/Linux-x86_64/510.60.02/NVIDIA-Linux-x86_64-510.60.02.run

**Comment by andrey-shostik** [comment]
Thank you. I have 8 RTX 3060 TI LHR V2, and it's working fine for several hours, but after, some cards falls down to 20 MH/S. Where can be the problem? 
![image](https://user-images.githubusercontent.com/13626849/167487737-c761f7d8-509a-47c4-86b2-3b009bc8c1e1.png)

**Reply by amusleh-spotware-com** [reply]
> Thank you. I have 8 RTX 3060 TI LHR V2, and it's working fine for several hours, but after some cards falls down to 20 MH/S. Where can be a problem? 
> ![image](https://user-images.githubusercontent.com/13626849/167487737-c761f7d8-509a-47c4-86b2-3b009bc8c1e1.png)
> 

İt's a known issue, lots of us have the same issue.

**Reply by s0laria** [reply]
Lowering the mem clock helps it a little, but the only stable solution for me is simply autorebooting every hour. Perhaps new version will be more stable.

**Comment by JuanCruzLopezz** [comment]
I can't reach 60 MH on my 3060 Ti, no matter what overclock I set it to, it won't go above 53 MH.

**Comment by sunshineyour** [comment]
ERROR - CUDA Error: an illegal memory access was encountered (err_no=700)
ERROR - Device 1 exception, exit ...
is bug?

**Comment by henryx167** [comment]
Hello,

I Have a problem with NBMiner after upgrade to new driver and new miner version.

My current parameters are:
NBMiner 41.0
Nvidia driver: 510.60.02

System info:
INFO - ------------------- System -------------------
INFO - OS:     Ubuntu 18.04.6 LTS, 5.10.0-hiveos
INFO - CPU:                  Intel(R) Pentium(R) D CPU 3.00GHz
INFO - RAM:    3212 MB / 3924 MB
INFO - CU_DRV: 11.6, 510.60.02

GPU info:
=== GPU 0, 04:00.0 GeForce RTX 3060 12288 MB ===
  Bios 94.06.14.80.9B, PCIE Link Gen 1, PCIE Link Width 16x
  UUID GPU-3ad5224b-fb77-717d-b229-e32a0431ddf8, ROM flash: Not supported
  Power Limit:  Min 100.0 W, Default 170.0 W, Max 180.0 W, Current 170.0 W
  Frequency:  GFXCore 210 MHz, MEMCore 405 MHz
  Memory:  Total 12288 MB, Used 255 MB, Free 12032 MB, Samsung GDDR6
  Utilization:  GPU 0 %, MEM 20 %, Throttle: Idle
  PSTATE P8, PWR 16.0 W, Temp 44 °C, Fan 0 %, RPM 0, RPM 0

When the miner starts I see this ERROR
-------------------------------------------------------------------------------------------
ERROR - Devices initialize failed.
ERROR - No GPU available for mining.
ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
ERROR - Mining program unexpected exit.
ERROR - Code: 255, Reason: Unknown error
ERROR - Restart miner after 10 secs ...
ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-------------------------------------------------------------------------------------------

![error_nbimer](https://user-images.githubusercontent.com/68892285/168430231-00aa8c7e-6197-4132-9586-6bcb1fff652d.jpg)

Can you advise me what this means and how to fix it?

Thank you.

**Comment by O-Ogi** [comment]
> `feature`: `ethash` 100% LHR unlocker added, for both Windows & Linux.
> 
> - Run nbminer with admin priviledge to get 100% LHR unlock
> - Tested and verified on drivers: `512.15` for Windows, `510.60` for Linux.
> - Other driver versions may have some compatibility issue.
> - Previous LHR mode are removed.
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/NebuTech/NBMiner/releases/tag/v41.0'>v41.0</a>.</em>

**Reply by O-Ogi** [reply]
a077e1d208b19a1eb2c010911e26ac6cc6079364

**Reply by O-Ogi** [reply]
> a077e1d208b19a1eb2c010911e26ac6cc6079364
